scripts/partner_batch_imports.py

```diff
@@ -161,7 +164,12 @@ def update_state(logfile, fname, line_num=0):
 
 def csv_to_ol_json_item(line):
     """converts a line to a book item"""
-    b = Biblio(line.strip().split('|'))
+    try:
+        data = line.strip().split('|')
```

Comment:
doesn't this line need a `decode()` to make it a string from UTF-8 bytes?   (otherwise split will error because it wants a `b'|'`)

```
b'test|stuff'.decode().split('|')
['test', 'stuff']

```

vs.
```
b'test|stuff'.split('|')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: a bytes-like object is required, not 'str'
```